actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -69,78 +69,43 @@ module TranslationHelper
       # resolved against.
       #
       def translate(key, **options)
-        unless options[:default].nil?
-          remaining_defaults = Array.wrap(options.delete(:default)).compact
-          options[:default] = remaining_defaults unless remaining_defaults.first.kind_of?(Symbol)
-        end
+        return key.map { |k| translate(k, **options) } if key.is_a?(Array)
 
-        # If the user has explicitly decided to NOT raise errors, pass that option to I18n.
-        # Otherwise, tell I18n to raise an exception, which we rescue further in this method.
-        # Note: `raise_error` refers to us re-raising the error in this method. I18n is forced to raise by default.
-        if options[:raise] == false
-          raise_error = false
-          i18n_raise = false
-        else
-          raise_error = options[:raise] || ActionView::Base.raise_on_missing_translations
-          i18n_raise = true
+        defaults = if options.key?(:default)
+          options[:default].is_a?(Array) ? options[:default].compact : [options[:default]]
```

Comment:
How about removing `default` from the options hash here, as we did before?

```suggestion
          options[:default].is_a?(Array) ? options.delete(:default).compact : [options.delete(:default)]
```

That way when `default` is the only option provided, we still hit the `options.empty?` fast path in `html_escape_translation_options`. I suspect passing `default` as the only option is a pretty common case.

Reply:
Good suggestion! :+1: